iphone/Classes/TiUtils.m

```diff
@@ -794,8 +794,13 @@ +(NSURL*)toURL:(NSString *)relativeString relativeToURL:(NSURL *)rootPath
 			}
 		}
 	}
-
-	result = [NSURL URLWithString:[relativeString stringByAddingPercentEscapesUsingEncoding:NSUTF8StringEncoding] relativeToURL:rootPath];
+	//only add percentescape if there are spaces in relativestring
+	if ([[relativeString componentsSeparatedByString:@" "] count] -1 == 0) {
```

Comment:
This will work for the fix but it is not the right fix. For example if the image was specified as 'image%1.png' this if condition will fail and you would get a nil result. 

The real problem is in ImageView loadUrl function where we are using 
`NSURL *url_ = [TiUtils toURL:[img absoluteString] proxy:self.proxy];`

What we sould be using is 
`NSURL *url_ = [TiUtils toURL:[img path] proxy:self.proxy];`